sdk/appconfiguration/Azure.ResourceManager.AppConfiguration/src/Azure.ResourceManager.AppConfiguration.csproj

```diff
@@ -9,4 +9,8 @@
     <PackageTags>azure;management;appconfiguration</PackageTags>
     <NoWarn>$(NoWarn);AZC0001;AZC0008;</NoWarn>
   </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.AutoRest.CSharp" VersionOverride="$(MgmtAutorestVersion)" PrivateAssets="All" />
```

Comment:
Is there a reason we need to manually add this into all the projects as opposed to overriding in https://github.com/Azure/azure-sdk-for-net/blob/master/eng/Packages.Data.props? YOu could add a condition for the mgmt library in there and set the version you want there instead of needing to override this in all the projects.

Reply:
Because we don't want it to be a permanent state.  As each project upgrades to the new autorest this will be removed

Reply:
Ok if the plan is to undo this in the near future that is fine either approach will work. However you could have saved yourself some file edits by putting it in the common Packages.Data.props file.